xml/System.Windows.Forms/Control.xml

```diff
@@ -3602,6 +3604,67 @@ The following table lists Windows Forms controls and which event (<xref:System.W
         <altmember cref="T:System.Windows.Forms.BindingManagerBase" />
       </Docs>
     </Member>
+    <Member MemberName="DataContext">
+      <MemberSignature Language="C#" Value="public virtual object? DataContext { get; set; }" />
+      <MemberSignature Language="ILAsm" Value=".property instance object DataContext" />
+      <MemberSignature Language="DocId" Value="P:System.Windows.Forms.Control.DataContext" />
+      <MemberSignature Language="VB.NET" Value="Public Overridable Property DataContext As Object" />
+      <MemberSignature Language="F#" Value="member this.DataContext : obj with get, set" Usage="System.Windows.Forms.Control.DataContext" />
+      <MemberSignature Language="C++ CLI" Value="public:&#xA; virtual property System::Object ^ DataContext { System::Object ^ get(); void set(System::Object ^ value); };" />
+      <MemberType>Property</MemberType>
+      <AssemblyInfo>
+        <AssemblyName>System.Windows.Forms</AssemblyName>
+        <AssemblyVersion>7.0.0.0</AssemblyVersion>
+      </AssemblyInfo>
+      <Attributes>
+        <Attribute FrameworkAlternate="windowsdesktop-7.0">
+          <AttributeName Language="C#">[System.ComponentModel.Bindable(true)]</AttributeName>
+          <AttributeName Language="F#">[&lt;System.ComponentModel.Bindable(true)&gt;]</AttributeName>
+        </Attribute>
+        <Attribute FrameworkAlternate="windowsdesktop-7.0">
+          <AttributeName Language="C#">[System.ComponentModel.Browsable(false)]</AttributeName>
+          <AttributeName Language="F#">[&lt;System.ComponentModel.Browsable(false)&gt;]</AttributeName>
+        </Attribute>
+      </Attributes>
+      <ReturnValue>
+        <ReturnType>System.Object</ReturnType>
+      </ReturnValue>
+      <Docs>
+        <summary>To be added.</summary>
```

Comment:
@RussKie Will your team bring in missing doc comments before .NET 7 ships?

Reply:
@singhashish-wpf and @pchaurasia14 did you port docs for WPF?

Reply:
I'm going to resolve this conversation just so I can merge the PR. I believe that Carlos is going to port the docs in a separate PR.